src/main/scala/io/sdkman/changelogs/JavaMigrations.scala

```diff
@@ -957,4 +957,27 @@ class JavaMigrations {
     Seq(Linux64, MacOSX, Windows).foreach(removeVersion("java", "13.ea.19-open", _))
   }
 
+  @ChangeSet(order = "138", id = "138-add_graalvm_19_0_0", author = "ilopmar")
+  def migrate138(implicit db: MongoDatabase) = {
+    List(
+      Version(
+        candidate = "java",
+        version = "19.0.0-graal-ce",
```

Comment:
We'll need to stick to using the same `-grl` suffix as before in order for the _sdkman-hooks_ service to handle these versions.

Reply:
Pushed the change. Feel free to squash the commits when merging them

Reply:
Thanks @ilopmar. You have some serious Scala skills there!!! :laughing:

Reply:
Sorry for being a pain, it needs to be `-grl` without the `-ce` for the hooks to be picked up.

Reply:
I'm not so sure, 3 commits to make a simple change :(

Reply:
You did well, thanks for doing this! :metal: